tasks/server_keys.yml

```diff
@@ -1,156 +1,163 @@
   when: openvpn_tls_auth_key is not defined
 
 # not a security issue, params aren't secret, just not generated by an attacker
 # per http://security.stackexchange.com/questions/42415/openvpn-dhparam/42418#42418
-- name: copy pre-generated DH params
+- name: Copy pre-generated DH params
   copy:
     src: dh.pem
-    dest: "{{openvpn_key_dir}}"
+    dest: "{{ openvpn_key_dir }}"
     owner: root
     group: root
     mode: 0400
   when: openvpn_use_pregenerated_dh_params|bool
 
 # Alternatively, if you're concerned about logjam attacks
-- name: generate dh params
-  command: openssl dhparam -out {{openvpn_key_dir}}/dh.pem {{openvpn_rsa_bits}}
+- name: Generate dh params
+  command: openssl dhparam -out {{ openvpn_key_dir }}/dh.pem {{ openvpn_rsa_bits }}
   args:
-    chdir: "{{openvpn_key_dir}}"
+    chdir: "{{ openvpn_key_dir }}"
     creates: dh.pem
   when: not (openvpn_use_pregenerated_dh_params|bool)
 
-- name: install ca.conf config file
+- name: Install ca.conf config file
   template:
     src: ca.conf.j2
-    dest: "{{openvpn_key_dir}}/ca.conf"
+    dest: "{{ openvpn_key_dir }}/ca.conf"
     owner: root
     group: root
-    mode: 744
+    mode: "1350"
 
-- name: create initial certificate revocation list squence number
+- name: Create initial certificate revocation list squence number
   shell: "echo 00 > crl_number"
   args:
-    chdir: "{{openvpn_key_dir}}"
+    chdir: "{{ openvpn_key_dir }}"
     creates: crl_number
 
-- name: generate tls-auth key
+- name: Generate tls-auth key
   command: openvpn --genkey --secret ta.key
   args:
-    chdir: "{{openvpn_key_dir}}"
+    chdir: "{{ openvpn_key_dir }}"
     creates: ta.key
   when: openvpn_tls_auth_key is not defined
 
-- name: install revocation script
+- name: Install revocation script
   template:
     src: revoke.sh.j2
-    dest: "{{openvpn_key_dir}}/revoke.sh"
+    dest: "{{ openvpn_key_dir }}/revoke.sh"
     owner: root
     group: root
-    mode: 744
+    mode: "1350"
 
-- name: check if certificate revocation list database exists
+- name: Check if certificate revocation list database exists
   stat:
-    path: "{{openvpn_key_dir}}/index.txt"
+    path: "{{ openvpn_key_dir }}/index.txt"
   register: file_result
 
-- name: create certificate revocation list database if required
+- name: Create certificate revocation list database if required
   file:
-    path: "{{openvpn_key_dir}}/index.txt"
+    path: "{{ openvpn_key_dir }}/index.txt"
     state: touch
-  when: file_result.stat.exists == False
+    mode: "0644"
+  when: not file_result.stat.exists
 
-- name: set up certificate revocation list
+- name: Set up certificate revocation list
   command: sh revoke.sh
   args:
     chdir: "{{ openvpn_key_dir }}"
     creates: "{{ openvpn_key_dir }}/ca-crl.pem"
 
-- name: install crl-cron script
+- name: Install crl-cron script
   template:
     src: crl-cron.sh.j2
     dest: "{{ openvpn_base_dir }}/crl-cron.sh"
     owner: root
     group: root
-    mode: 744
+    mode: "1350"
```

Comment:
`u=rw,g=r,o=r` would probably be better. 

1350 is 744 decimal in octal, so it was wrong in the first place based on https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html#parameter-mode

Reply:
The reason I kept it 1350 was to not introduce any change in the files installed by the older versions. But you are right, I'll update this.